web/web.go

```diff
@@ -1117,23 +1117,23 @@ func rateSchema(cols, warnings int64, missingPKey bool) string {
 	ok := func(total, badCount int64) bool { return badCount < total/3 }
 	switch {
 	case cols == 0:
-		return "GRAY"
+		return "NONE"
 	case warnings == 0 && !missingPKey:
-		return "GREEN"
+		return "EXCELLENT"
 	case warnings == 0 && missingPKey:
-		return "BLUE"
+		return "GOOD"
 	case good(cols, warnings) && !missingPKey:
-		return "BLUE"
+		return "GOOD"
 	case good(cols, warnings) && missingPKey:
-		return "BLUE"
+		return "GOOD"
 	case ok(cols, warnings) && !missingPKey:
```

Comment:
What is yellow here ? We should replace that too

Reply:
Yellow was associated with "OK" according to the previous implementation

Reply:
Yellow is replaced with "OK"